main/src/cgeo/geocaching/filters/core/StatusGeocacheFilter.java

```diff
@@ -224,6 +229,12 @@ public void setStatusSolvedMystery(final Boolean statusSolvedMystery) {
         this.statusSolvedMystery = statusSolvedMystery;
     }
 
+    public Boolean getStatusHasUserDefineWaypoint() { return statusHasUserDefineWaypoint; }//Volthai7us
```

Comment:
Code does not follow c:geo Codacy/checkstyle rules (see breaking Codacy run), those need to be fixed. 

I recommend you install the "checkstyle" plugin in your Android studio, run it using the c:geo configuration (checkstyle.xml) and fix all findings before pushing to server. This might save you many round-trips :-)

Reply:
ok now, thank you